packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/MrwSerializationTypeDefinitions/TestData/SerializationCustomizationTests/CanChangePropertyName/Model.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Sample.Models
 {
-    public partial class MockInputModel
+    public partial class Model
```

Comment:
I was running into build errors on my local due to this file path being too long. There's probably another overall fix we can do for that, but for now I'm shortening the filename.